Monika After Story/game/script-story-events.rpy

```diff
@@ -507,6 +507,8 @@ label calendar_birthdate:
     m 1eud "So [player], when were you born?"
     call mas_bday_player_bday_select_select
     $ mas_stripEVL('mas_birthdate',True)
+    $ mas_HKBDropShield()
```

Comment:
`mas_DropShield_dlg` runs in `call_next_event`, and that covers every hkb case except music, so to specifically address that, should just call `mas_MUMUDropShield` right after pushing this event.